crates/sui-types/src/move_package.rs

```diff
@@ -88,6 +90,22 @@ pub struct UpgradeCap {
     pub policy: u8,
 }
 
+/// Rust representation of `sui::package::UpgradeTicket`.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct UpgradeTicket {
+    pub cap: ID,
```

Comment:
what is this field?

Reply:
Maybe I'm just a fool and should look at the `sui::package` code lol

Reply:
https://github.com/MystenLabs/sui/blob/73dcf13605dcbec42646b75451245794946da155/crates/sui-framework/sources/package.move#L57-L76